examples/docs_snippets/docs_snippets/guides/dg/migrating-definitions/8-list-defs-after-all.txt

```diff
@@ -1,6 +1,6 @@
 dg list defs
 
-
+Using /Users/smackesey/stm/code/elementl/oss/examples/docs_snippets/.tox/docs_snapshot_update/bin/dagster-components
```

Comment:
These are a result of changed default venv resolution behavior in the existing project case and will be fixed in a docs PR upstack